### DISCUSSION TITLE
Problems with export Excel

### DISCUSSION THREAD

**Discussion by amisano** [dbeaver/dbeaver#23330] [General]
Hello everyone from Enrico in Italy
I use this addon for export to excel:
https://dbeaver.io/update/office/latest/
https://marketplace.eclipse.org/content/dbeaver-office-integration

I have been using this addon for several years with great satisfaction but I have a problem in the excels that are created.
At the end of each Excel column, after the field value, unnecessary spaces are left which create problems for me.
I use this tool to migrate data to a new ERP which has problems if the spaces at the bottom are empty.
Can you help me understand why this happens?
Is this the standard operation of the office integration or is it a bug?
Thanks so much for the help

I have added an excel file for example

[DIMMP10V_202404061313.xlsx](https://github.com/dbeaver/dbeaver/files/14893367/DIMMP10V_202404061313.xlsx)

**Comment by LonwoLonwo** [comment]
Hello @amisano 

Did you try to enable the "trim strings" setting on the format setting page during the export?
